src/styles/styles.js

```diff
@@ -159,6 +159,14 @@ const styles = {
         color: themeColors.textReversed,
     },
 
+    fixedBottomButton: {
+        position: 'absolute',
```

Comment:
I'm not sure if position: absolute is the best way to implement this. Shouldn't we use flexbox instead? This way the button container will stay anchored at the bottom, and any UI above the button will be in a flex container that can scroll. Otherwise we might have issues where the button container appears on top of the content behind it, and you can't scroll to access that UI that is indeed blocked by the button.

Reply:
You can easily see how this doesn't work well by just making your window shorter:
![dkJFPEh2fL](https://user-images.githubusercontent.com/2319350/111595887-e16c9580-87cc-11eb-94ec-56022044134f.gif)

So I think we need something like this:
````
<flex container direction=column>
    <flex grow=1>
        Content for profile view
    </flex>
    <flex grow=0>
        Save Button
    </flex>
</flex>

Reply:
Makes sense, I'll update the button styling